Countdown timer for server & client

DelCherub opened the pull request:
- The game now waits until all players have loaded the game to begin.
- When all players have loaded, it starts a countdown timer, and then the game only starts once the timer has expired.
- The countdown is managed in LevelManager and the value/state of the countdown can be acted upon in GameplayState or RunningState.

DelCherub committed:
Player waiting & countdown

- The server now waits until all players are loaded before starting to update;
- The server will initiate a countdown when all players have loaded, and will only start the game when the countdown is over.

DelCherub committed:
Relocated countdownTimer to levelManager

-Moved countdownTimer to level manager as that is where the other timers are, and it makes more logical sense.

DelCherub committed:
Countdown now exists client-side

-Client now has a countdown which should match up with the server, which could be used for UI, sounds etc.

DelCherub committed:
Added cout for testing

DelCherub committed:
Removed test cout

AdSand reviewed (dismissed):
Tested in multiplayer and works

DelCherub committed:
Fisex playrs being able to input in countdown

- Players can no longer jump in the countdown
- Fixed bug with jump at the start of the game where jumpVelocity was way too high.
